mindsdb/integrations/handlers/writer_handler/question_answer.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, args: dict, model_parameters: ModelParameters):
         retriever = get_retriever(
             embeddings_model_name=self.embeddings_model_name,
             persist_directory=self.persist_directory,
-            collection_name=args.get('collection_name', '_default_collection')
+            collection_name=args.get("collection_name", "langchain"),
```

Comment:
collection name must be alphanumeric only

Reply:
Perhaps let's add a check or mention it in the handler's `README`? Not a blocker for merging, though,